src/Http/Cookie/Cookie.php

```diff
@@ -251,7 +271,7 @@ public function getName()
      */
     protected function validateName($name)
     {
-        if (preg_match("/[=,; \t\r\n\013\014]/", $name)) {
+        if (preg_match("/[=,;\t\r\n\013\014]/", $name)) {
```

Comment:
@markstory do you know of any documented list of invalid characters? If so, we could link to it with a ``@link``.